CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 * NetworkedMessageChannels can now be subscribed to before initiating a connection (#670)
 * Refactored connection management into simpler state machine (#666)
 * Merged GameState bridge classes (the ones that contained no or limited functionality) (#697) This cleans up our sometimes too verbose code split.
+* Configured the NetworkTransform components of every NetworkObject to reduce the bandwidth usage (#716). This prevents the unnecessary synchronization of data that clients do not need, i.e. a character's scale or y position.
```

Comment:
It could be interesting to give a quick bandwidth stat. Like "we went from x to y kb/s in this z scenario"

Reply:
I'd still be interested in the bandwidth usage, in addition to the character sizes. At the end of the day, it's total bandwidth we care about.